packages/data-point/lib/entity-types/base-entity/factory.js

```diff
@@ -29,6 +61,12 @@ function create (Factory, spec, id) {
     entity.error = createReducer(spec.error)
   }
 
+  const inputType = getTypeModifier(spec.inputType)
+  entity.inputType = createReducer(inputType)
+
+  const outputType = getTypeModifier(spec.outputType)
+  entity.outputType = createReducer(outputType)
```

Comment:
if `spec.inputType` or `spec.outputType` are not defined, I don't think we should add reducers for them with `acceptAnyType` (it's just another thing to resolve with no added benefit). Something like this should work because `resolveReducer` makes sure the reducer is truthy before resolving it:

```js
if (spec.outputType) {
  const outputType = getTypeModifier(spec.outputType)
  entity.outputType = createReducer(outputType)
}
```

https://github.com/ViacomInc/data-point/blob/master/packages/data-point/lib/reducer-types/resolve.js#L28